packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -527,8 +527,12 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   final bool enableInteractiveSelection;
 
-  /// Setting this property to true makes the cursor stop blinking and stay visible on the screen continually.
-  /// This property is most useful for testing purposes.
+  /// Setting this property to true makes the cursor stop blinking and stay
+  /// visible on the screen continually. This property is most useful for
+  /// testing purposes.
+  ///
+  /// It doesn't affect when the cursor is first drawn (on focus of the
```

Comment:
This sentence is unclear to me. What does it mean?

Reply:
ha, PTAL attempt number 2